11-algorithms.Rmd

```diff
@@ -161,9 +161,9 @@ It helps to further break down this approach into discrete tasks before writing
 4. Find the area-weighted mean of triangle centroids\index{centroid}.
 
 These steps may sound straightforward, but converting words into working code requires some work and plenty of trial-and-error, even when the inputs are constrained:
-The algorithm will only work for *convex polygons*, which contain no internal angles greater than 180°, no star shapes allowed (packages **decido** and **sfdct** can triangulate non-convex polygons using external libraries, as shown in the [algorithm](https://geocompr.github.io/geocompkg/articles/algorithm.html) vignette at geocompr.github.io). 
+The algorithm will only work for *convex polygons*, which contain no internal angles greater than 180°, no star shapes allowed (packages **decido** and **sfdct** can triangulate non-convex polygons using external libraries, as shown in the [algorithm](https://geocompx.github.io/geocompkg/articles/algorithm.html) vignette hosted at [geocompx.github.io](https://geocompx.github.io/). 
```

Comment:
geocompx.org

Reply:
Where is that article on geocompx.org though @Nowosad ?